public/sw.js

```diff
@@ -17,18 +17,31 @@ self.addEventListener('install', event => {
   });
 });
 
-// versioned JS files
-const cacheableFiles = [
-  'https://mybbc-analytics.files.bbci.co.uk/reverb-client-js/reverb-3.9.2.js',
-  'https://mybbc-analytics.files.bbci.co.uk/reverb-client-js/smarttag-5.29.4.min.js',
-];
-
 const fetchEventHandler = async event => {
-  if (
+  const cacheableFiles = [
```

Comment:
Doesn't really matter but if this is a static list could it be moved out of the function and defined as `CACHABLE_FILES` if its immutable?

Reply:
It sure can, thanks!